tests/converter/rasa_sample_project/policies.yml

```diff
@@ -0,0 +1,9 @@
+policies:
+  - name: KerasPolicy
```

Comment:
Do we need these policies? They seem to refer to Keras code, which is not supported by Mindmeld.

Reply:
If you ever want to run the sample rasa project to a demo or something, I think you would need this file. Or are you asking if MindMeld needs them somehow?

Reply:
Yeah this is just the Rasa config.